Source/utils/autofill_array.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <array>
+
+namespace devilution {
+
+template<typename T, std::size_t N>
+struct AutofillArray: public std::array<T, N>
+{
+	using std::array<T, N>::array;
+
+	AutofillArray(T x)
```

Comment:
Single-argument constructors should be marked `explicit` https://google.github.io/styleguide/cppguide.html#Implicit_Conversions

Reply:
Not in this case. The whole point was to make MissileSpriteData legible.

Reply:
You can do something like `{15}` in `MissileSpriteData`

Reply:
Wasn't needed; the initializer list used the explicit constructor anyway. You get a bogus-sounding error if you try to put braces around it.